src/core/xref.js

```diff
@@ -72,6 +84,72 @@ export async function run(conf, elems) {
   addDataCiteToTerms(results, xrefMap, conf);
 }
 
+function invalidProfileError(profile) {
+  pub(
+    "error",
+    `Invalid \`xref\` profile option passed in the configuration: "${profile}". Please use one of the supported specification profiles: ${Object.keys(
```

Comment:
Nit: split this message into multiple lines.

Reply:
`\n` before "Please use one of the..." would work?

Reply:
Sorry I wasn't clear. I mean, the line is too long (more than 80 characters), so split it and join multiple lines using `+`.

Reply:
Ah, my bad, you meant too long in the source code, right? Will do.